app_dart/bin/server.dart

```diff
@@ -69,6 +70,7 @@ Future<void> main() async {
         githubChecksService: githubChecksService,
         scheduler: scheduler,
       ),
+      '/api/query-github-graphql': QueryGithubGraphql(config, authProvider),
```

Comment:
Can you add api user doc here. I'm trying to add more documentation, and closest to where the api is defined seems like a good place.

Implementation doc can be added at the handler class.

Reply:
Note, that dart doc style says to add doc near the class def, but for api's I think it makes it more readable to have them intertwined here. But your call on where to add it.